src/lipizzaner_master.py

```diff
@@ -126,8 +126,11 @@ def _start_experiments(self):
             self.experiment_id = db_logger.create_experiment(self.cc.settings)
             self.cc.settings['general']['logging']['experiment_id'] = self.experiment_id
 
+        client_id = 0
```

Comment:
Use `for i, client in enumerate(self.cc.settings['general']['distribution']['client_nodes']):` to skip the separate counting

Reply:
Sure!